src/test/java/org/gridsuite/filter/server/ExpertFilterUtilsTest.java

```diff
@@ -1,560 +0,0 @@
-
-        boolean result = andFilter.evaluateRule(gen);
-
-        assertTrue(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterWithFalseANDCombination() {
-        List<AbstractExpertRule> andRules1 = new ArrayList<>();
-        NumberExpertRule numRule1 = NumberExpertRule.builder().value(0.0)
-                .field(FieldType.MIN_P).operator(OperatorType.LOWER).build();
-        andRules1.add(numRule1);
-        NumberExpertRule numRule2 = NumberExpertRule.builder().value(105.0)
-                .field(FieldType.MAX_P).operator(OperatorType.GREATER_OR_EQUALS).build(); // false
-        andRules1.add(numRule2);
-        CombinatorExpertRule andCombination = CombinatorExpertRule.builder().combinator(CombinatorType.AND).rules(andRules1).build();
-
-        List<AbstractExpertRule> andRules2 = new ArrayList<>();
-        andRules2.add(andCombination);
-        NumberExpertRule numRule3 = NumberExpertRule.builder().value(20.0)
-                .field(FieldType.TARGET_V).operator(OperatorType.EQUALS).build();
-        andRules2.add(numRule3);
-        EnumExpertRule enumRule4 = EnumExpertRule.builder().value("HYDRO")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.EQUALS).build();
-        andRules2.add(enumRule4);
-
-        CombinatorExpertRule andFilter = CombinatorExpertRule.builder().combinator(CombinatorType.AND).rules(andRules2).build();
-
-        boolean result = andFilter.evaluateRule(gen);
-
-        assertFalse(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterWithORCombination() {
-        List<AbstractExpertRule> orRules1 = new ArrayList<>();
-        NumberExpertRule numRule1 = NumberExpertRule.builder().value(0.0)
-                .field(FieldType.MIN_P).operator(OperatorType.LOWER).build();
-        orRules1.add(numRule1);
-        NumberExpertRule numRule2 = NumberExpertRule.builder().value(100.0)
-                .field(FieldType.MAX_P).operator(OperatorType.GREATER_OR_EQUALS).build(); // false
-        orRules1.add(numRule2);
-        CombinatorExpertRule orCombination = CombinatorExpertRule.builder().combinator(CombinatorType.OR).rules(orRules1).build();
-
-        List<AbstractExpertRule> orRules2 = new ArrayList<>();
-        orRules2.add(orCombination);
-        NumberExpertRule numRule3 = NumberExpertRule.builder().value(20.0)
-                .field(FieldType.TARGET_V).operator(OperatorType.EQUALS).build();
-        orRules2.add(numRule3);
-        EnumExpertRule enumRule4 = EnumExpertRule.builder().value("OTHER")
-                .field(FieldType.ENERGY_SOURCE).operator(OperatorType.EQUALS).build(); //false
-        orRules2.add(enumRule4);
-
-        CombinatorExpertRule orFilter = CombinatorExpertRule.builder().combinator(CombinatorType.OR).rules(orRules2).build();
-
-        boolean result = orFilter.evaluateRule(gen);
-
-        assertTrue(result);
-    }
-
-    @Test
-    void testEvaluateExpertFilterWithFalseORCombination() {
```

Comment:
Same

Reply:
Idem, rule tree is tested by https://github.com/gridsuite/filter-server/pull/90/files#diff-b4e0eb7fb6e066e9822531fb175401d400663c91679b6a7c2a936fe2fd7e74ccR48